src/game/engine/coord.h

```diff
@@ -192,7 +233,22 @@ inline DirType Cell_Direction8(cell_t cell1, cell_t cell2)
     return Desired_Facing8(Cell_Get_X(cell1), Cell_Get_Y(cell1), Cell_Get_X(cell2), Cell_Get_Y(cell2));
 }
 
-int Distance(coord_t coord1, coord_t coord2);
+DirType Direction(coord_t coord1, coord_t coord2);
+DirType Direction(target_t target1, target_t target2);
+DirType Direction(cell_t cell1, cell_t cell2);
+
+int Coord_Distance(coord_t coord1, coord_t coord2);
```

Comment:
Why rename the Distance functions to have distinct names rather than rely on overload when you don't do the same for Direction?